### ISSUE TITLE
checkpoint_converter.py | error in function write_weights ...?

### ISSUE THREAD

**Issue by Oriolla** [magenta/magenta-js#565]
In checkpoint_converter.py:

write_weights([entries],
    output_dir,
     write_manifest=True,
    **quantization_dtype**=quantization_dtype,
    <span class="pl-s1">shard_size_bytes</span><span class="pl-c1">=</span><span class="pl-s1">shard_mb</span> <span class="pl-c1">*</span> <span class="pl-c1">1024</span> <span class="pl-c1">*</span> <span class="pl-c1">1024</span>)

Is it right? May be **quantization_dtype_map**?

**Comment by damjanprvc** [comment]
Yes, had the same issue. Changed to 'quantization_dtype_map' and worked fine. 
Opened a new pull request as well.
